internal/providers/terraform/azure/notification_hubs.go

```diff
@@ -0,0 +1,102 @@
+package azure
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetAzureRMNotificationHubsRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_notification_hub_namespace",
+		RFunc: NewAzureRMNotificationHubs,
+	}
+}
+
+func NewAzureRMNotificationHubs(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var monthlyAdditionalPushes *decimal.Decimal
+	var startUsageAmt string
+
+	if u != nil && u.Get("monthly_additional_pushes").Type != gjson.Null {
+		monthlyAdditionalPushes = decimalPtr(decimal.NewFromInt(u.Get("monthly_additional_pushes").Int()))
+	}
+
+	sku := "Basic"
+	location := d.Get("location").String()
+
+	if d.Get("sku_name").Type != gjson.Null {
+		sku = d.Get("sku_name").String()
+	}
+	costComponents := make([]*schema.CostComponent, 0)
+
+	costComponents = append(costComponents, notificationHubsCostComponent("Base Charge Per Namespace", location, sku))
+	if monthlyAdditionalPushes.GreaterThan(decimal.NewFromInt(10000000)) {
+		startUsageAmt = "10"
+		message := "Over 10M"
+		multi := 10000000
+		if sku == "Standard" && monthlyAdditionalPushes.GreaterThan(decimal.NewFromInt(10000000)) {
+			message = "10-100M"
+		}
+		if sku == "Standard" && monthlyAdditionalPushes.GreaterThan(decimal.NewFromInt(1000000000)) {
+			startUsageAmt = "100"
+			multi = 1000000000
+			message = "Over 100M"
+		}
+		costComponents = append(costComponents, notificationHubsPushesCostComponent(fmt.Sprintf("Additional Pushes (%s)", message), location, sku, startUsageAmt, monthlyAdditionalPushes, multi))
```

Comment:
Sorry for the nitpick, we try to capitalize only the first letter unless it's an acronym. The consistency is important so users get a familiar output regardless of the IaC tool or cloud they use...

```suggestion
		costComponents = append(costComponents, notificationHubsPushesCostComponent(fmt.Sprintf("Additional pushes (%s)", message), location, sku, startUsageAmt, monthlyAdditionalPushes, multi))
```